Added lease quota count and rate limit count docs (#22509)
hashicorp/vault: website/content/api-docs/system/lease-count-quotas.mdx

  concept of roles (such as `/auth/approle/`), this will make the quota restrict login
  requests to that mount that are made with the specified role. The request will fail if
  the auth mount does not have a concept of roles, or `path` is not an auth mount.
- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,

>   all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to

Reviewer: ```suggestion
  all child namespaces of the specified path namespace will inherit the quota. The `inheritable` parameter cannot be set to
```

Style correction: avoid "this" as a pronoun

>   true if the `path` does not specify a namespace. All root namespace quotas are inheritable by default.
>   All other namespace quotas are not inheritable by default.

Reviewer: ```suggestion
  `true` if the `path` does not specify a namespace. Only the quotas associated
  with the root namespace are inheritable by default.
```

  concept of roles (such as `/auth/approle/`), this will make the quota restrict login
  requests to that mount that are made with the specified role. The request will fail if
  the auth mount does not have a concept of roles, or `path` is not an auth mount.

> - `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,

Reviewer 2: ```suggestion
- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a non-root namespace,
```

Reviewer 2: Wait, actually, this doesn't make sense. Removing this suggestion.

all child namespaces of the specified path namespace will inherit this quota. The request
will fail if `inheritable` is set to true, while the `path` has a value that is not a namespace mount,

> or if a value for `role` is specified. All global quotas are inheritable by default. All namespace quotas

Reviewer 2: I think I'd suggest rewording this for clarity, especially as we made the decision that root quotas can't be made uninheritable:
"All global quotas are inheritable by default. All namespace quotas are not inheritable by default."
-> 
"Root namespace quotas are inheritable by default and this cannot be changed, thereby making them global in reach. All other namespace quotas are not inheritable by default."

> all child namespaces of the specified path namespace will inherit this quota. The request
> will fail if `inheritable` is set to true, while the `path` has a value that is not a namespace mount,
> or if a value for `role` is specified. All global quotas are inheritable by default. All namespace quotas

Reviewer 2: ```suggestion
all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to true if the path does not specify a namespace. All global quotas are inheritable by default. All namespace quotas
```

Reviewer 2: Suggested rewording of the "This will fail" section. In particular, `role` can never be applied on a namespace path anyway
